ecl/dns/v2/recordset.py

```diff
@@ -28,6 +28,8 @@ class RecordSet(resource2.Resource):
     allow_list = True
     allow_get = True
 
+    _query_mapping = resource2.QueryParameters("zone_id", "marker", "limit")
```

Comment:
zone_id is not query parameter. So I don't think we require it here. Please confirm it again.
```suggestion
    _query_mapping = resource2.QueryParameters("marker", "limit")
```